scripts/quests/otherAreas/Elderly_Pursuits.lua

```diff
@@ -0,0 +1,157 @@
+-----------------------------------
+-- Elderly Pursuits
+-----------------------------------
+-- Log ID: 4, Quest ID: 75
+-----------------------------------
+--Pre-requisite Completed Secrets of Oven Lost Log ID: 4, Quest ID: 73
+-----------------------------------
+-- Despachiaire !gotoid 16883721
+-- Rouva !gotoid 17719492
+--- Para NM ??? !gotoid 16785771
+-----------------------------------
+local ID = zones[xi.zone.CARPENTERS_LANDING]
+-----------------------------------
```

Comment:
`---` is for the Lua language server, otherwise comments are `-- ` (with the space)

Reply:
Done.